src/beam.cpp

```diff
@@ -293,8 +293,9 @@ void BeamSegment::CalcBeamInit(
     }
 
     // Only if not only rests. (Will produce non-sense output anyway)
-    if (m_beamElementCoordRefs.size() != nbRests) {
-        this->m_avgY /= (m_beamElementCoordRefs.size() - nbRests);
+    const auto elementCount = static_cast<decltype(m_avgY)>(m_beamElementCoordRefs.size());
```

Comment:
Would a cast to `(int)` just work? I know this might be more correct, but I don't think we are going to ever have problems with `.size()` cast to `int`. This looks overly complicated, and I don't like seeing `auto` in such places because it makes it harder to understand what the type actually is.

Reply:
Ok. done.